curved_bottom_navigation/src/main/java/np/com/susanthapa/curved_bottom_navigation/CurvedBottomNavigationView.kt

```diff
@@ -753,21 +846,49 @@ class CurvedBottomNavigationView @JvmOverloads constructor(
     }
 
     override fun onDraw(canvas: Canvas) {
-        super.onDraw(canvas)
+
         if (!isMenuInitialized) {
             return
         }
+
+
+
+
+        // Draw circle for FAB (Selected item indicator)
         canvas.drawCircle(centerX, curCenterY, fabSize / 2f, fabPaint)
-        // draw the AVD within the circle
+
+        // Draw AVD (Animated Vector Drawable) within the FAB circle
         menuAVDs[fabIconIndex].setBounds(
             (centerX - menuIcons[fabIconIndex].intrinsicWidth / 2).toInt(),
             (curCenterY - menuIcons[fabIconIndex].intrinsicHeight / 2).toInt(),
             (centerX + menuIcons[fabIconIndex].intrinsicWidth / 2).toInt(),
             (curCenterY + menuIcons[fabIconIndex].intrinsicHeight / 2).toInt()
         )
+
         menuAVDs[fabIconIndex].draw(canvas)
-        // draw the path last so that we can clip the FAB and AVD
+
+        // Draw the path for the bottom navigation
         canvas.drawPath(path, navPaint)
+
+        // Loop through each bottom navigation item
+        for (i in bottomNavItemViews.indices) {
+            val itemCenterX = menuCellWidth * i + (menuCellWidth / 2f)
+            val textY = layoutHeight - 8 // Position the text
+
+            // If the item is selected, show a dot instead of text
+            if (i == selectedIndex) {
+                canvas.drawCircle(itemCenterX, layoutHeight -12, dotRadius, dotPaint)
```

Comment:
Do you think it is possible to make the dot optional? I think having a title is nice but about the dot, I'm not sure if everyone wants it.